saleor/order/error_codes.py

```diff
@@ -31,3 +31,4 @@ class OrderErrorCode(Enum):
     DUPLICATED_INPUT_ITEM = "duplicated_input_item"
     NOT_AVAILABLE_IN_CHANNEL = "not_available_in_channel"
     CHANNEL_INACTIVE = "channel_inactive"
+    UNAVAILABLE_VARIANT_IN_CHANNEL = "unavailable_variant_in_channel"
```

Comment:
We have `NOT_AVAILABLE_IN_CHANNEL`, are these two error code different or do they refer to the same error? Now I noticed that we have the function `validate_variant_channel_listings` - again, isn't it the same functionality as we are adding here?